_data/social.yml

```diff
@@ -36,7 +36,7 @@ websites:
       sms: Yes
       software: Yes
       exceptions:
-          text: "SMS required for 2FA. 1 account per phone number, 1 phone number per account."
+          text: "SMS required for 2FA setup, but OATH generator/Facebook mobile app can be added afterwards."
```

Comment:
Instead of changing this to `SMS required for 2FA setup, but OATH generator/Facebook mobile app can be added afterwards.`, simply remove the message about one phone number per account.

The line should look like this:
`        text: "SMS required for 2FA. 1 account per phone number."`